src/plugins/vis_types/timeseries/public/application/components/lib/index_pattern_select/switch_mode_popover.tsx

```diff
@@ -116,7 +116,7 @@ export const SwitchModePopover = ({ onModeChange, useKibanaIndices }: PopoverPro
           label={i18n.translate(
             'visTypeTimeseries.indexPatternSelect.switchModePopover.useKibanaIndices',
             {
-              defaultMessage: 'Use only index patterns',
+              defaultMessage: 'Use Kibana data views',
```

Comment:
I'd be great if this label changed based on the current state:

- Use data views
- Query Elasticsearch indices

Reply:
I think that this will be confusing unless I don't get the suggestion correctly. Imagine you are using the data view mode and the switch is on, you are switching off the toggle to use the string mode. If the label is `Query Elasticsearch indices` but the toggle is off it will be very confusing.

Reply:
That's fair. Feel free to ignore this feedback.